src/components/SignUp/Landing.tsx

```diff
@@ -175,6 +176,15 @@ const Landing: React.FC<{
           </Row>
         </Col>
       </Row>
+      <Row>
+        <TaglineTwo>
```

Comment:
This line probably wouldn't be classified as a Tagline - I'd assume it's just a regular `p` with maybe some unique styles for this context, such as being italic, having the margin-top, etc. We don't really need any of the tagline stuff you made for this IMO, just semantically